client/src/pages/Home.js

```diff
@@ -59,7 +59,7 @@ const Home = () => {
 								/>
 							</div>
 							<div className="CopyButton-div">
-								<CopyButton text={outputValue} />
+								<CopyButton text={alternativeValue} />
 							</div>
```

Comment:
@Lexi, there are additional changes required for fixing the copy button bug. Can I request that we assign this task to Hadi?

Reply:
it's currently working when you click on the copy button in the alternative box, but yeah if you have other bugs to fix, please do!